appconfiguration/azure-appconfiguration-conversion-sample-complete/pom.xml

```diff
@@ -12,8 +12,8 @@
 
   <artifactId>spring-cloud-azure-appconfiguration-convert-sample-complete</artifactId>
   <groupId>com.azure.spring</groupId>
-  <version>2.0.0</version> <!-- {x-version-update;com.azure.spring:azure-spring-cloud-appconfiguration-convert-sample-complete;dependency} -->
-  <name>Azure App Configuration Conversion Sample</name>
+  <version>1.0.0</version>
+  <name>Convert Cosmos DB Application Using App Configuration and Key Vault Complete</name>
```

Comment:
And

Reply:
Actually, I hope to keep the same, 
there are 11 `and` in the output, I think it's better to keep them the same
![image](https://user-images.githubusercontent.com/4465723/148524371-58ff5586-3b83-47af-abc3-bb5a138c3661.png)